### PR TITLE
OCLOMRS-571: Raise Test Coverage

### PR DESCRIPTION
**Pull request by myrdstom**
# JIRA TICKET NAME:
[Raise Test Coverage](https://issues.openmrs.org/browse/OCLOMRS-571)

# Summary:
Address the falling coverage on this [PR](https://github.com/openmrs/openmrs-ocl-client/pull/478)

### CODE REVIEW

File: src/tests/bulkConcepts/components/Pagination.test.js [hunk 1]
```diff
@@ -119,6 +119,28 @@ describe('Test suite for Paginations component', () => {
     wrapper.find('.fa-angle-double-right').simulate('click');
     expect(wrapper.find('Paginations').instance().props.currentPage + 1).toEqual(2);
   });
+  it('it should go to previous page', () => {
```
Comment (dkayiwa) [inline src/tests/bulkConcepts/components/Pagination.test.js]: What should go?

Comment (myrdstom) [reply src/tests/bulkConcepts/components/Pagination.test.js]: I have made the description clearer

File: src/tests/bulkConcepts/components/Pagination.test.js [hunk 2]
```diff
@@ -119,6 +119,28 @@ describe('Test suite for Paginations component', () => {
     wrapper.find('.fa-angle-double-right').simulate('click');
     expect(wrapper.find('Paginations').instance().props.currentPage + 1).toEqual(2);
   });
+  it('should call setPreviousPage ', () => {
```
Comment (dkayiwa) [inline src/tests/bulkConcepts/components/Pagination.test.js]: What should call set previous page?

Comment (myrdstom) [reply src/tests/bulkConcepts/components/Pagination.test.js]: I have been more specific with the description

File: src/tests/bulkConcepts/components/Pagination.test.js (hunk 1 above)
Comment (myrdstom) [reply src/tests/bulkConcepts/components/Pagination.test.js]: I have made adjustments to the description

File: src/tests/bulkConcepts/components/Pagination.test.js (hunk 2 above)
Comment (myrdstom) [reply src/tests/bulkConcepts/components/Pagination.test.js]: I have made adjustments to the description

**Pull request merged**